biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -22,92 +30,128 @@ export const InputPage = () => {
   const [startDate, setStartDate] = useState(new Date());
   const [endDate, setEndDate] = useState(new Date());
   const [smallBatteryCells, smallBatteryCellInput] = useInput({
-    type: 'number',
+    default: 0,
   });
   const [largeBatteryCells, largeBatteryCellInput] = useInput({
-    type: 'number',
+    default: 0,
   });
   const [numBuildings, numBuildingsInput] = useInput({
-    type: 'number',
+    default: 0,
   });
   const [numSolarPanels, numSolarPanelsInput] = useInput({
-    type: 'number',
+    default: 0,
   });
-  const [townSize, townSizeInput] = useInput({
-    type: 'number',
+  const [townWidth, townWidthInput] = useInput({
+    default: 0,
+  });
+  const [townHeight, townHeightInput] = useInput({
+    default: 0,
   });
   const history = useHistory();
 
   const onSubmit = async (e: React.SyntheticEvent<EventTarget>) => {
     e.preventDefault();
-    const body = {
+    function serialize(obj: { [k: string]: unknown }) {
+      const str = [];
+      for (const p in obj)
+        str.push(
+          encodeURIComponent(p) +
+            '=' +
+            encodeURIComponent(
+              (obj[p] as { toString: () => string }).toString()
+            )
+        );
+      return str.join('&');
+    }
+
+    const params = {
       startDate,
       endDate,
-      smallBatteryCells,
-      largeBatteryCells,
-      numBuildings,
-      numSolarPanels,
-      townSize,
-
+      smallBatteryCells: smallBatteryCells,
+      largeBatteryCells: largeBatteryCells,
+      numBuildings: numBuildings,
+      numSolarPanels: numSolarPanels,
+      townHeight: townHeight,
+      townWidth: townWidth,
```

Comment:
You shouldn't need to explicitly set the values here since they're the same name